src/Index/nctPage.jsx

```diff
@@ -114,6 +114,10 @@ class IndexPageComponent extends React.Component {
                 </Button>
               </form>
             </div>
+            <div className='high-light nct-right-box'>
+              <p><strong>Data Sharing and Submission</strong></p>
+              <p>If you are interested in making your data available via this platform, please <a href='mailto:accessclinicaldatasupport@niaid.nih.gov'>contact us.</a></p>
```

Comment:
do you think we should try to get the email from the config `components.login.email` so we can update it in a single place if needed?